examples/colors/README.md

```diff
@@ -0,0 +1,8 @@
+# Colors
+
+This demo displays a grid of colors.
+It is using a GtkGridView, and shows how to display and sort the data in various ways. The controls for this are implemented using GtkDropDown.
+The dataset used here has up to `16777216` items.
+Note that this demo also functions as a performance test for some of the list model machinery, and the biggest sizes here can lock up the application for extended times when used with sorting.
```

Comment:
Worth mentioning it's a Rust rewrite of a gtk4-demo written in C & link to it's source code if possible